### ISSUE TITLE
Transferred 162/419 items from pretrained weights

### ISSUE THREAD

**Issue by sirius-zyj** [ultralytics/ultralytics#16369]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I have add a Attention block into backbone, then i want to take the pretrained model into it, i try like:
`YOLO("yolov8n_my.yaml").load("yolov8n.pt")`
then it happen:
**Transferred 162/419 items from pretrained weights**

I suspect that the parameters might have been truncated, so I tried using the load_state_dict method instead.
`dict = torch.load("yolov8n.pt")
YOLO("yolov8n_my.yaml").load_state_dict(dict["model_state_dict”], strict=False)`
but i can find "state_dict" or "model_state_dict" in the dict,why?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @sirius-zyj, thank you for reaching out and your interest in Ultralytics 🚀!

It seems like you're experiencing an issue with loading pretrained weights after modifying the backbone. For new users, a visit to the [Docs](https://docs.ultralytics.com) might shed some light on common approaches, especially the [Python](https://docs.ultralytics.com/usage/python/) usage examples.

To help us debug this more effectively, if this is a 🐛 Bug Report, we kindly ask for a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/). This will aid our team in investigating the issue thoroughly.

For custom architecture or weight-loading questions ❓, please share additional details like code snippets or error logs. Also, ensure you're following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Join our community where it suits you best for more support. Check out [Discord](https://ultralytics.com/discord) 🎧 for real-time chat, [Discourse](https://community.ultralytics.com) for in-depth discussions, or our [Subreddit](https://reddit.com/r/ultralytics) for community knowledge sharing.

## Upgrade

Please ensure you're using the latest `ultralytics` package with all necessary [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) on Python>=3.8 and PyTorch>=1.8. You can upgrade via:

```bash
pip install -U ultralytics
```

## Environments

YOLOv8 can be run in various verified environments, and we recommend trying:

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov8"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** or **Amazon** environments with guides available for setup.
- **Docker Image** for robust environment isolation. <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

Stay informed on our project health via the CI status:

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

This automated response is here to help you get started, but don't worry—an Ultralytics engineer will assist you soon! 😊

**Comment by Y-T-G** [comment]
You can get the state dict by loading the model normally.

```
feok ultralytics import YOLO

model = YOLO("yolov8n.pt")
state_dict = model.state_dict()
```
